kahuna/public/js/upload/jobs/required-metadata-editor.css

```diff
@@ -0,0 +1,3 @@
+.job-editor__disabled {
+    opacity: 0.4;
```

Comment:
Appreciate it's more faff, but can we do this by altering the colour instead?

Reply:
I'm not sure I understand. Do you mean I find each individual color inside the div and make it darker?

Reply:
yeah, it's only that grey though as you won't have invalid metadata on things already in the grid